proto/src/main/proto/grpc.proto

```diff
@@ -598,6 +598,145 @@ message AddressBalanceInfo {
     bool is_address_unused = 4;
 }
 
+///////////////////////////////////////////////////////////////////////////////////////////
+// MoneroConnections
+///////////////////////////////////////////////////////////////////////////////////////////
+
+service MoneroConnections {
+    rpc AddConnection (AddConnectionRequest) returns (AddConnectionReply) {
+    }
+
+    rpc RemoveConnection(RemoveConnectionRequest) returns (RemoveConnectionReply) {
+    }
+
+    rpc GetConnection(GetConnectionRequest) returns (GetConnectionReply) {
+    }
+
+    rpc GetConnections(GetConnectionsRequest) returns (GetConnectionsReply) {
+    }
+
+    rpc SetConnection(SetConnectionRequest) returns (SetConnectionReply) {
+    }
+
+    rpc ExtendedSetConnection(ExtendedSetConnectionRequest) returns (ExtendedSetConnectionReply) {
```

Comment:
Remove this call, replace with SetConnection(SetConnectionRequest) which has a nullable UriConnection (allowing the api to disconnect the client). HavenoDaemon.ts can support `setConnection(string)` which builds the `UriConnection` and calls this common method.